.changeset/calm-rivers-behave.md

```diff
@@ -0,0 +1,5 @@
+---
+'@livekit/components-core': minor
```

Comment:
```suggestion
'@livekit/components-core': patch
```

as it's primarily a bugfix, I think we should mark it as a patch update

Reply:
Fixed, ready for merge!